mlflow/server/js/src/common/utils/Utils.js

```diff
@@ -184,6 +184,16 @@ class Utils {
     }
   }
 
+  /**
+   * Get the duration of a run given start- and end time.
+   *
+   * @param startTime in milliseconds
+   * @param endTime in milliseconds
+   */
+  static getDuration(startTime, endTime) {
+    return startTime && endTime ? this.formatDuration(endTime - startTime) : null;
```

Comment:
Awesome! Could we make a call to this utility in `RunView` as well? https://github.com/mlflow/mlflow/blob/e25910bd5bb750222dd9c316ad132b99b66c707e/mlflow/server/js/src/experiment-tracking/components/RunView.js#L206-L207

Reply:
Great suggestion, added it now